Intro (#51)
infinum/eightshift-docs: website/docs/basics/basics.md

title: Basics
sidebar_label: Basics
---

> Before WordPress landed in 5.0 release it way mostly PHP based project with some JavaScript and CSS in the themes. Not it it a bit different, we now have PHP in the core, JavaScript (specifically Reach) in the Block Editor and some styles in the editor part and the theme part.

Reviewer: ```suggestion
Before WordPress landed in 5.0 release, the themes were mostly PHP based project with some JavaScript and CSS. Now it's a bit different, because  now we have PHP in the core, JavaScript (specifically React.js) in the Block Editor and some styles in the editor and in the theme parts of the website.
```

> Now that everything is different we must adopt to the change and as well.

Reviewer: ```suggestion
Now that everything is different, we must adopt to the change and as well.
```

> As you all-ready read in the previous chapters about [Eightshift Libs](eightshift-libs.md) and [Eightshift Frontend Libs](eightshift-frontend-libs.md) we have two totally different technologies in out boilerplate so we must separate the documentation the same way.

Reviewer: ```suggestion
As you already saw in the previous chapters about [Eightshift Libs](eightshift-libs.md) and [Eightshift Frontend Libs](eightshift-frontend-libs.md), we have two totally different technologies in out boilerplate, so we must separate the documentation the same way.
```

> ## Lets finally start with documentation

Reviewer: ```suggestion
## Lets' finally start with documentation
```

We took our time to setup 40+ WP-CLI commands that will help you setup everything and use all our features without worrying about correct names, files and organisation. You can find out more on this link:

[WP-CLI](wp-cli)

> If you want to know how to use service classes, or how Dependency injection container works with autowiring, or maybe you would like to read about our extensive WP-CLI features, this link is for you:

Reviewer: ```suggestion
If you want to know how to use service classes, or how dependency injection container works with autowiring, or maybe you would like to read about our extensive WP-CLI features, this link is for you:
```

[Backend stuff](backend)

> Or you would rather check what we have prepared regarding the JavaScript Babel, Webpack, or how we ensure tha all our code is the same across multiple developers, then this link is for you:

Reviewer: ```suggestion
If you would rather check what we have prepared regarding the JavaScript, Babel, Webpack, or how we ensure that all our code is the same across multiple developers, then this link is for you:
```

[Frontend stuff](frontend)

> On the other hand if you would like to see or extensive **Dynamic Blocks** collection and how you can use them with all the developers flexibility and as little code duplication as possible, go check this link:

Reviewer: ```suggestion
On the other hand, if you would like to see our extensive **Dynamic Blocks** collection and how you can use them with all the developers flexibility and as little code duplication as possible, go check this link:
```

## Let's finally start with documentation

> We took our time to set up bunch WP-CLI commands that will help you set up everything and use all our features without worrying about correct names, files, and organization. You can find out more on this link:

Reviewer 2: ```suggestion
We took our time to set up a bunch of WP-CLI commands. That should help you set up everything and use all our features, without worrying about correct names, files, and organization. You can find out more about it on this link:
```
